### ISSUE TITLE
CONTRIBUTORS.txt breaks installation of 4.3

### ISSUE THREAD

**Issue by nedbat** [coveragepy/coveragepy#542]
*Originally reported by* **Margus Kerma (Bitbucket: [marguskerma](https://bitbucket.org/marguskerma), GitHub: Unknown)**

----------------------------------------

```
Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-tdp_tggr/coverage/setup.py", line 50, in <module>
        paras = contributors.read().split("\n\n")
      File "/usr/lib/python3.5/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xc4 in position 846: ordinal not in range(128)
```

Non ascii chars in CONTRIBUTORS.txt cause the install to fail on Linux. Tested on Ubuntu 16.04.

Can be quickly verified with docker: `docker run --rm -it zeroturnaround/python3 pip3 install coverage==4.3`

[Dockerfile source](https://bitbucket.org/zeroturnaround/docker-python3/src/tip/Dockerfile) 

----------------------------------------
- Bitbucket: https://bitbucket.org/ned/coveragepy/issue/542

**Comment by nedbat** [comment]
*Original comment by* **Reinout van Rees (Bitbucket: [reinout](https://bitbucket.org/reinout), GitHub: [reinout](https://github.com/reinout))**

----------------------------------------

Verified, a colleague has the same problem.

Ah, I see this is fixed already. https://bitbucket.org/ned/coveragepy/commits/502d5e2aad49

And that this bug report is a duplicate of #540, so it can be closed.

**Comment by nedbat** [comment]
Duplicate of #540.

**Comment by nedbat** [comment]
*Original comment by* **Margus Kerma (Bitbucket: [marguskerma](https://bitbucket.org/marguskerma), GitHub: Unknown)**

----------------------------------------

Thanks for quick response. Issue solved in 4.3.1.
